source/content/drupal-launch-check.md

```diff
@@ -36,15 +36,6 @@ To generate the reports, Pantheon uses [Site Audit](https://drupal.org/project/
 
 ## Frequently Asked Questions
 
-### Deprecated Configuration for the Temporary Files Path Warning on Drupal 8.8.0
```

Comment:
Hi @greg-1-anderson ! Now that OTTO-72 is closed, should we reopen this PR to remove this note about the warning?